### PR TITLE
Made default (column) names consistent following tidyverse style guide

### LINKED ISSUE #6291: Make default (column) names consistent following tidyverse style guide
When a dialog produces a new column/graph/model/object/data frame, we give a default new name. The names should follow the the [tidyverse style guide](http://style.tidyverse.org/syntax.html).

In particular, we need someone to go through the dialogs are check and correct the default names for a few things:
- Names should be all lower case e.g. "Date" needs to be "date"
- Separate words with underscore (not . or -) e.g. "my_col"
- Avoid long names. This is hard because names should also be meaningful. I think we should aim for 7-8 characters max where possible.

@maxwellfundi @rdstern Can you suggest someone to take this task?

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #6291
This is to make the default name when a dialog produces a new column/graph/model/object/data frame following tidyverse style guide column name.
@lloyddewit @dannyparsons there is no coding in this PR and this is ready to be merge. Thanks.

### CODE REVIEW

File: instat/dlgExtremesClimatic.vb [hunk 1]
```diff
@@ -272,7 +272,7 @@ Public Class dlgExtremesClimatic
         ucrNudDeclusterColumns.SetRDefault("1")
 
         ucrSaveDeclusteredPlot.SetDataFrameSelector(ucrSelectorClimaticExtremes.ucrAvailableDataFrames)
-        ucrSaveDeclusteredPlot.SetPrefix("declustredplot")
+        ucrSaveDeclusteredPlot.SetPrefix("declustred_plot")
```
Comment (shadrackkibet) [inline instat/dlgExtremesClimatic.vb:275]: `declustered_plot`?

File: instat/dlgInfillMissingValues.vb [hunk 2]
```diff
@@ -397,21 +397,21 @@ Public Class dlgInfillMissingValues
 
     Private Sub ucrPnlMethods_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlMethods.ControlValueChanged, ucrReceiverElement.ControlValueChanged
         If rdoNaApproximate.Checked Then
-            ucrSaveNewColumn.SetPrefix("Int_" & ucrReceiverElement.GetVariableNames(False))
+            ucrSaveNewColumn.SetPrefix("tnt_" & ucrReceiverElement.GetVariableNames(False))
```
Comment (shadrackkibet) [inline instat/dlgInfillMissingValues.vb:400]: `int_`

File: instat/dlgOtherRosePlots.vb [hunk 3]
```diff
@@ -270,15 +270,15 @@ Public Class dlgOtherRosePlots
     Private Sub ucrPnlOptions_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlOptions.ControlValueChanged
         ucrReceiverDate.SetMeAsReceiver()
         If rdoPolarAnnulus.Checked Then
-            ucrSaveGraph.SetPrefix("Polar_Annulus")
+            ucrSaveGraph.SetPrefix("polar_annulus")
```
Comment (shadrackkibet) [inline instat/dlgOtherRosePlots.vb:273]: Seems we always add `plot/graph`? so `polar_annulus_plot`? Same case below.

File: instat/dlgParallelCoordinatePlot.vb [hunk 4]
```diff
@@ -100,7 +100,7 @@ Public Class dlgParallelCoordinatePlot
         ucrInputScale.SetRDefault("std")
         ucrInputScale.SetDropDownStyleAsNonEditable()
 
-        ucrSaveGraph.SetPrefix("ParCoord")
+        ucrSaveGraph.SetPrefix("par_coord")
```
Comment (shadrackkibet) [inline instat/dlgParallelCoordinatePlot.vb:103]: `par_coord_plot`?

File: instat/dlgTaylorDiagram.vb [hunk 5]
```diff
@@ -69,7 +69,7 @@ Public Class dlgTaylorDiagram
         ucrChkNormalise.SetText("Normalise")
         ucrChkNormalise.SetRDefault("FALSE")
 
-        ucrSavePlot.SetPrefix("taylordiagram")
+        ucrSavePlot.SetPrefix("taylor_diagram")
```
Comment (shadrackkibet) [inline instat/dlgTaylorDiagram.vb:72]: add `plot`

File: instat/dlgConditionalQuantilePlot.vb [hunk 6]
```diff
@@ -129,7 +129,7 @@ Public Class dlgConditionalQuantilePlot
         ucrReceiverStatistics.SetLinkedDisplayControl(lblStatistics)
         ucrReceiverType.SetLinkedDisplayControl(lblType)
 
-        ucrSavePlot.SetPrefix("conditionalquantile")
+        ucrSavePlot.SetPrefix("cond_quantile")
```
Comment (shadrackkibet) [inline instat/dlgConditionalQuantilePlot.vb:132]: `conditional_quantile_plot`?

File: instat/dlgConditionalQuantilePlot.vb [hunk 7]
```diff
@@ -216,14 +216,14 @@ Public Class dlgConditionalQuantilePlot
     Private Sub ucrPnlOptions_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlOptions.ControlValueChanged
         If rdoQuantiles.Checked Then
             ucrBase.clsRsyntax.SetBaseRFunction(clsConditionalQuantileFunction)
-            ucrSavePlot.SetPrefix("conditionalquantile")
+            ucrSavePlot.SetPrefix("cond_quantile")
```
Comment (shadrackkibet) [inline instat/dlgConditionalQuantilePlot.vb:219]: `conditional_quantile_plot`

File: instat/dlgConditionalQuantilePlot.vb [hunk 8]
```diff
@@ -216,14 +216,14 @@ Public Class dlgConditionalQuantilePlot
...
             Me.Size = New System.Drawing.Size(Me.Width, 489)
             ucrBase.Location = New Point(ucrBase.Location.X, 397)
             ucrSavePlot.Location = New Point(ucrSavePlot.Location.X, 370)
             cmdPlotOptions.Text = "Conditional Quantile Options"
         Else
             ucrBase.clsRsyntax.SetBaseRFunction(clsConditionalEvalFunction)
-            ucrSavePlot.SetPrefix("conditionalEval")
+            ucrSavePlot.SetPrefix("cond_eval")
```
Comment (shadrackkibet) [inline instat/dlgConditionalQuantilePlot.vb:226]: `conditional_eval_plot`

**Review by shadrackkibet — CHANGES_REQUESTED** [review]
Some suggestions

File: instat/dlgConditionalQuantilePlot.vb (hunk 8 above)
Comment (shadrackkibet) [reply instat/dlgConditionalQuantilePlot.vb:226]: perhaps `cond_eval_plot`? is that too long?

Comment (N-thony) [reply instat/dlgConditionalQuantilePlot.vb:226]: it is too long but the meaning is clear, I think it is okay.

Comment (N-thony) [reply instat/dlgConditionalQuantilePlot.vb:226]: it is too long but the meaning is clear, I think it is okay.

**Comment by N-thony** [comment]
@shadrackkibet I made the corrections on your above comments. please have a look. Thanks.

**Comment by shadrackkibet** [comment]
@rdstern ?

**Review by rdstern — APPROVED** [review]

**Review by shadrackkibet — APPROVED** [review]

**Pull request merged**